docs/generating-tests.rst

```diff
@@ -0,0 +1,417 @@
+
+==========================
+Generating Consensus Tests
+==========================
+
+.. warning:: This guide targets Linux users.  It might work on Mac OS X.  It will probably not work on Windows.
+
+
+Consensus Tests
+===============
+
+This article describes writing tests with the C++ Ethereum client (Aleth). Consensus tests 
+are test cases for all Ethereum implementations. The test cases are distributed 
+in the "filled" form, which contains, for example, the expected state root hash after transactions.
+The filled test cases are usually not written by hand, but generated from "test filler" files.
+``testeth`` executable in cpp-ethereum can convert test fillers into test cases.
```

Comment:
cpp-ethereum has been renamed to aleth there are some places in this document which still mentions cpp-ethereum.

Reply:
Thanks Hugo @hugo-dc, have updated all remaining references.